package.json

```
@@ -93,11 +94,14 @@
     "test:esm": "c8 mocha --enable-source-maps ./test/esm/*.mjs --check-leaks",
     "coverage": "c8 report --check-coverage",
     "prepare": "npm run compile",
-    "pretest": "npm run compile -- -p tsconfig.test.json && cross-env NODE_ENV=test npm run build:cjs",
-    "compile": "rimraf build && tsc",
+    "pretest": "npm run compile && tsc -p tsconfig.test.json && cross-env NODE_ENV=test npm run build:cjs",
+    "compile": "rimraf build && tsc -p tsconfig.json",
+    "compile:watch": "npm run compile -- --watch",
     "postcompile": "npm run build:cjs",
     "build:cjs": "rollup -c rollup.config.cjs",
+    "build:cjs:watch": "rollup -w -c rollup.config.cjs",
     "postbuild:cjs": "rimraf ./build/index.cjs.d.ts",
+    "dev": "npm run compile && run-p compile:watch build:cjs:watch",
```

I think the current scripts mean this will run compile twice, and invokes rimraf twice. Maybe do the rimraf up front and remove it from the watch scripts. (Disclaimer: making these suggestions online without trying it for real!)
```suggestion
    "dev": "rimraf build && run-p compile:watch build:cjs:watch",
```

---

What is `run-p`? (Perhaps run-parallel from some package you have installed?)